.github/PULL_REQUEST_TEMPLATE

```diff
@@ -0,0 +1,23 @@
+<!-- 
+Please explain WHAT you changed.
+
+The title should be descriptive, for example:
+
+* *Fixed a typo in the apikeypermissions.md page*
+* *Added the maximum number of domain whitelabels you can create to domains.md*
+* *Fixing the number of days a batch id is valid in scheduling_parameters.md*
+
+If this PR fixes an issue, please reference the issue number as well.
+
+Fill out this form in the body:
+-->
+
+**Description of the change**:
+
+If you have questions, please send an email [Sendgrid](mailto:dx@sendgrid.com), or file a Github Issue in this repository.
+
+<!-- 
```

Comment:
We appreciate this, but it's going to show up on every pull request. Can you please remove this comment? 

@ksigler7 -- Are you cool with that?

Reply:
@mbernier Sure, I'll go ahead and remove the comment if @ksigler7 is okay with it.

Reply:
@nishithshah2211 yeah, no worries on removing it! Thanks for asking!